cross/org.eclipse.cdt.build.crossgcc/plugin.xml

```diff
@@ -32,7 +34,7 @@
             targetTool="cdt.managedbuild.tool.gnu.cross.c.linker;cdt.managedbuild.tool.gnu.cross.cpp.linker;cdt.managedbuild.tool.gnu.archiver">
          <targetPlatform
                archList="all"
-               binaryParser="org.eclipse.cdt.core.ELF"
+               binaryParser="org.eclipse.cdt.core.GNU_ELF"
```

Comment:
It looks like the binary parser attribute is written out in the .cproject file of every created project.
While this in general would be good, for this particular change, it looks like all the existing projects will be stuck on org.eclipse.cdt.core.ELF until a manual change is done. Do you have any idea how to automatically migrate existing projects to the org.eclipse.cdt.core.GNU_ELF parser instead?

Reply:
Will think about this. Perhaps as a separate issue. It is easy to switch the active  binary parser in the CDT project settings UI.

Do you think there could be any reason why someone would NOT wish to update their parser? I think it may be wise to prompt for confirmation.

Cc: @jonahgraham

Reply:
Right now, I do not see any reason why you would not want that.

Maybe we should invert the question and rather ask us why anyone would like to use the org.eclipse.cdt.core.ELF parser in the first place and if we have no clear answer to that, perhaps the org.eclipse.cdt.core.ELF and org.eclipse.cdt.core.GNU_ELF should be the same parser from now on?

Reply:
There are, of course, non-GNU toolchains that generate ELF executable files. At present, the existence of the ELF parser and the GNU_ELF parser allows common code to determine whether use of GNU tools such as `objcopy` should be attempted. In theory, an extender could integrate equivalent non-GNU tools.